index.html

```diff
@@ -211,17 +210,28 @@ <h2> LET's CONNECT</h2>
 
 <div class = "separator">
     <img src="./img/skills/Separator Bottom.svg"></div>
-
-
-
 </ul>
-
-
 </div>
 </div>
+<div class="contact-me">
+    <h1>Contact me</h1>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form id ="form" action="https://formspree.io/projects/2121707390096113225" method="post">
+    <input id = "fname" type="text" name = "text" placeholder="Enter Your Name" required minlength="1" maxlength="30">
+    
+    <input id = "femail" type="email" name = "email" placeholder="Enter Your Email " required minlength="1" maxlength="30">
+    
+    <textarea id = "ftext" name = "form-text" placeholder="Write your message here"  maxlength="500"></textarea>
+    
+    <button id = "fbutton" class="button">Get in Touch</button>
+        </div>
+    
+    </form>
```

Comment:
- [ ] **Good job on using formspree to handle your contact form 👍 👍** however there is still an issue while submitting your form, Kindly, it would be better to fix this to make your project professional 😎😎😎

![image](https://user-images.githubusercontent.com/102757126/212966654-1e3fe7bb-f83f-4e1c-a070-681652dd3787.png)

Reply:
Fixed.